### PR TITLE
Slcf projects

### PR DESCRIPTION
**Pull request by ravinduf09**

### CODE REVIEW

File: web/src/Components/Programme/AddNewProgramme/programmeCreationComponent.tsx [hunk 1]
```diff
@@ -106,7 +106,7 @@ export const ProgrammeCreationComponent = (props: any) => {
   const [projectLocationMapOutlineLayer, setProjectLocationMapOutlineLayer] = useState<any>();
   const [projectLocationMapCenter, setProjectLocationMapCenter] = useState<number[]>([]);
 
-  const mapType = process.env.REACT_APP_MAP_TYPE ? process.env.REACT_APP_MAP_TYPE : 'None';
+  const mapType = process.env.REACT_APP_MAP_TYPE ? process.env.REACT_APP_MAP_TYPE : '';
```
Comment (tharindugayanga) [inline web/src/Components/Programme/AddNewProgramme/programmeCreationComponent.tsx:109]: keep 'None' here

File: web/src/Components/Maps/mapboxComponent.tsx [hunk 2]
```diff
@@ -31,6 +31,7 @@ export const MapboxComponent = (props: MapComponentProps) => {
 
   mapboxgl.accessToken = accessToken;
 
+  console.log('-------location props---------', layer, mapSource, outlineLayer);
```
Comment (tharindugayanga) [inline web/src/Components/Maps/mapboxComponent.tsx:34]: remove console log

File: web/src/Components/Programme/AddNewProgramme/programmeCreationComponent.tsx (hunk 1 above)
Comment (ravinduf09) [reply web/src/Components/Programme/AddNewProgramme/programmeCreationComponent.tsx]: added None

File: web/src/Components/SLCFProgramme/AddNewProgramme/SLCFProgrammeCreationComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,962 @@
...
+      }
+      count += 1;
+      lat += l[0];
+      long += l[1];
+    }
+    return [lat / count, long / count];
+  };
+
+  useEffect(() => {
+    console.log('-------project locations--------', projectLocations);
```
Comment (tharindugayanga) [inline web/src/Components/SLCFProgramme/AddNewProgramme/SLCFProgrammeCreationComponent.tsx:101]: remove

**Pull request merged**